zebra-chain/src/history_tree.rs

```diff
@@ -13,7 +15,7 @@ use crate::{
     block::{Block, ChainHistoryMmrRootHash, Height},
     orchard,
     parameters::{Network, NetworkUpgrade},
-    primitives::zcash_history::{Entry, Tree as InnerHistoryTree},
+    primitives::zcash_history::{Entry, Tree, V1, V2},
```

Comment:
I know we don't have any control over these names and are matching them, but it would be nice if they were something like `PreOrchard` and `PostOrchard` or something

Reply:
What do you think of 824f99d3? I've use `OrchardOnward` instead of `PostOrchard` to make clear it includes Orchard, and that's also how the spec names these cases

Reply:
Ooo nice! I like that